### PR TITLE
Fix #57948 - KeyError: 'platform.is_windows'

### LINKED ISSUE #57940: After upgrade to 3001, salt master throwing errors for network.py "name '__utils__' not found".
### Description of Issue
salt-master throwing errors after upgrade:
```
2020-07-14 17:19:48,376 [salt.loader      :870 ][CRITICAL][53069] Failed to load grains defined in grain file subnetZone.main in function <function main at 0x7f835763c8c8>, error:
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/salt/loader.py", line 859, in grains
    ret = funcs[key](**kwargs)
  File "/var/cache/salt/master/extmods/grains/subnetZone.py", line 317, in main
    grains['networkZone'] = findZone()
  File "/var/cache/salt/master/extmods/grains/subnetZone.py", line 219, in findZone
    internalips = salt.modules.network.ip_addrs(cidr='172.16.0.0/16')
  File "/usr/lib/python3.6/site-packages/salt/modules/network.py", line 1287, in ip_addrs
    addrs = __utils__["network.ip_addrs"](
NameError: name '__utils__' is not defined
```

### Setup
Fresh upgrade from 2019.2.0 to 3001, all done via rpm packages.

### Steps to Reproduce Issue
Errors appearing in `/var/log/salt/master`, new error messages several times a minute. 

### Versions Report
```
# salt --versions-report
Salt Version:
           Salt: 3001

Dependency Versions:
           cffi: Not Installed
       cherrypy: unknown
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
         Jinja2: 2.11.1
        libgit2: Not Installed
       M2Crypto: 0.35.2
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.2
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: Not Installed
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.6.8 (default, Sep 26 2019, 11:57:09)
   python-gnupg: Not Installed
         PyYAML: 3.13
          PyZMQ: 17.0.0
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.1.4

System Versions:
           dist: rhel 7.8 Maipo
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-1127.el7.x86_64
         system: Linux
        version: Red Hat Enterprise Linux Server 7.8 Maipo
```

### LINKED ISSUE #57948: [BUG] module/network.py uses platform.is_windows on Windows
**Description**
I have installed (via an update) version 3001 on various Windows machines. I am getting hourly log entries complaining about "platform.is_windows" inside modules/network.py.

**Setup**
Python 3 installation of salt-minion version 3001 on Windows (various OS versions). I have mine in pillar:

```
> mine_functions:
>   test.ping: []
>   network.ip_addrs: []
> 
```

**Steps to Reproduce the behavior**
Seems to be an automatic run hourly

**Log file entries**
```
2020-07-15 10:06:56,698 [salt.loader      :2000][ERROR   ][3260] Exception raised when processing __virtual__ function for salt.loaded.int.module.network. Module will not be loaded: 'platform.is_windows'
Traceback (most recent call last):
  File "c:\salt\bin\lib\site-packages\salt-3001-py3.7.egg\salt\loader.py", line 1983, in _process_virtual
    virtual = getattr(mod, virtual_func)()
  File "c:\salt\bin\lib\site-packages\salt-3001-py3.7.egg\salt\modules\network.py", line 38, in __virtual__
    if __utils__["platform.is_windows"]():
KeyError: 'platform.is_windows'
2020-07-15 10:06:56,698 [salt.loader      :2018][WARNING ][3260] salt.loaded.int.module.network.__virtual__() is wrongly returning `None`. It should either return `True`, `False` or a new name. If you're the developer of the module 'network', please fix this.
```

**Versions Report**
From the minion:
<details>
    Salt Version:
               Salt: 3001
     
    Dependency Versions:
               cffi: 1.12.2
           cherrypy: 17.4.1
           dateutil: 2.8.0
          docker-py: Not Installed
              gitdb: 2.0.5
          gitpython: Not Installed
             Jinja2: 2.10.1
            libgit2: Not Installed
           M2Crypto: Not Installed
               Mako: 1.0.7
       msgpack-pure: Not Installed
     msgpack-python: 0.5.6
       mysql-python: Not Installed
          pycparser: 2.19
           pycrypto: Not Installed
       pycryptodome: 3.9.7
             pygit2: Not Installed
             Python: 3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 20:34:20) [MSC v.1916 64 bit (AMD64)]
       python-gnupg: 0.4.4
             PyYAML: 5.1.2
              PyZMQ: 18.0.1
              smmap: 2.0.5
            timelib: 0.2.4
            Tornado: 4.5.3
                ZMQ: 4.3.1
     
    System Versions:
               dist:   
             locale: cp1252
            machine: AMD64
            release: 2012ServerR2
             system: Windows
            version: 2012ServerR2 6.3.9600 SP0
</details>

**Additional context**

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Uses `import salt.utils.platform` instead of relying on the function being present in `__utils__` in the `__virtual__` function for `modules/network.py`. Removes `__utils__` calls in other execution modules.

### What issues does this PR fix or reference?
Fixes: #57948 

### Previous Behavior
The minion was putting KeyError entries in the log file every time it ran anything.

### New Behavior
No more log file entries

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [x] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [x] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by markuskramerIgitt** [comment]
Interesting

File: salt/modules/boto_elbv2.py [hunk 1]
```diff
@@ -76,7 +72,7 @@ def __virtual__():
     """
     has_boto_reqs = salt.utils.versions.check_boto_reqs()
     if has_boto_reqs is True:
-        __utils__["boto3.assign_funcs"](__name__, "elbv2")
+        salt.utils.boto3mod.assign_funcs(__name__, "elbv2")
```
Comment (Ch3LL) [inline salt/modules/boto_elbv2.py:75]: Is `__utils__` not loaded in `__virtual__` in all cases? It looks like we do load that before modules.

**Review by Ch3LL — CHANGES_REQUESTED** [review]
looks like some tests will need to be written. oh also a changelog entry as described here https://docs.saltstack.com/en/master/topics/development/changelog.html will need to be added

File: salt/modules/boto_elbv2.py (hunk 1 above)
Comment (twangboy) [reply salt/modules/boto_elbv2.py:75]: I think it's a race condition...

**Comment by twangboy** [comment]
@Ch3LL What am I writing tests for on this one? The `__virtual__` function? All those other changes are for the py2 upgrade pre-commit stuff.

**Comment by Ch3LL** [comment]
yes you only need to add test coverage for the bug you are fixing.

File: salt/modules/boto_elbv2.py (hunk 1 above)
Comment (twangboy) [reply salt/modules/boto_elbv2.py:75]: Apparently `__utils__` is only loaded in the `__virtual__` function for state modules... not execution modules.

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**